### DISCUSSION TITLE
Music is playing entity

### DISCUSSION THREAD

**Discussion by JeroenV99** [hassio-addons/addon-spotify-connect#266] [Ideas]
I’m interested in adding a feature to the Spotify Connect integration that would provide an entity or attribute to indicate whether music is currently playing. This would help in automating the connection to a Bluetooth speaker when playback starts.

**Comment by dexterdar** [comment]
Yes, this would be helpful.  If someone else cast their Spotify music on the Spotify Connect, none of the Spotify Integration entities reflect that this add-on is playing.  Would be great to have an entity showing the state of Spotify Connect so it can be used in automations and conditions.
